documentation/migration-guides/Az.13.0.0-migration-guide.md

```diff
@@ -0,0 +1,1784 @@
+    - This change is expected to take effect from Az.Storage version: 8.0.0 and Az version: 13.0.0
+  - `-SrcFile`
+    - The type of parameter SrcFile will be changed from CloudFile to ShareFileClient. The alias CloudFile will be deprecated.
+    - This change is expected to take effect from Az.Storage version: 8.0.0 and Az version: 13.0.0
+  - `-SrcShare`
+    - The type of parameter SrcShare will be changed from CloudFileShare to ShareClient. The alias CloudFileShare will be deprecated.
+    - This change is expected to take effect from Az.Storage version: 8.0.0 and Az version: 13.0.0
+
+
+#### Before
+```powershell
+$f1 = Get-AzStorageFile -ShareName $sharename1 -Path $filepath -Context $ctx 
+$f2 = Start-AzStorageFileCopy -SrcFile $f1.CloudFile -DestShareName $shareName -DestFilePath $destfilepath -DestContext $ctx
+
+$f2.CloudFile
+
+$fdest = Get-AzStorageFile -ShareName $sharename2 -Path $filepath2 -Context $ctx
+Start-AzStorageFileCopy -SrcFile $f1.CloudFile -DestFile $fdest.CloudFile -Force 
+
+$share = Get-AzStorageShare -Name $sharename -Context $ctx
+Start-AzStorageFileCopy -SrcShare $s.CloudFileShare -SrcFilePath $srcfilepath -DestShareName $shareName -DestFilePath $destfilepath -DestContext $ctx
+```
+#### After
+```powershell
+$f1 = Get-AzStorageFile -ShareName $sharename1 -Path $filepath -Context $ctx 
+$f2 = Start-AzStorageFileCopy -SrcFile $f1.ShareFileClient -DestShareName $shareName -DestFilePath $destfilepath -Force -DestContext $ctx
+
+$f2.ShareFileClient
+
+$fdest = Get-AzStorageFile -ShareName $sharename2 -Path $filepath2 -Context $ctx
+Start-AzStorageFileCopy -SrcFile $f1.ShareFileClient -DestFile $fdest.ShareFileClient -Force -DestContext $ctx
+
+$share = Get-AzStorageShare -Name $sharename -Context $ctx
+Start-AzStorageFileCopy -SrcShare $s.ShareClient -SrcFilePath $srcfilepath -DestShareName $shareName -DestFilePath $destfilepath -Force -DestContext $ctx
+```
+
+
+### `Stop-AzStorageFileCopy`
+
+- Parameter breaking-change will happen to all parameter sets
+  - `-File`
+    - The parameter File (alias CloudFile) will be deprecated, and ShareFileClient will be mandatory.
+    - This change is expected to take effect from Az.Storage version: 8.0.0 and Az version: 13.0.0
+
+
+#### Before
+```powershell
+$fd = Get-AzStorageFile -ShareName $shareName -Path $filepath -Context $ctx
+Stop-AzStorageFileCopy -File $fd.CloudFile -Context $ctx -Force
+```
+#### After
+```powershell
+$fd = Get-AzStorageFile -ShareName $shareName -Path $filepath -Context $ctx
+Stop-AzStorageFileCopy -ShareFileClient $fd.ShareFileClient -Context $ctx -Force
+```
+
+
+### `Get-AzStorageBlobContent`
+- Parameter breaking-change will happen to parameter set UriPipeline
+  - When download blob with parameter AbsoluteUri (alias Uri, BlobUri), parameter Context will not be allowed to input together.
+  - This change is expected to take effect from Az.Storage version: 8.0.0 and Az version: 13.0.0
```

Comment:
So we need this sentence for every breaking parameter? 
As this file title has indicate this is Az.13.0.0 migration guild, we might don't need to mention it again and again. So the file can be easier to read.

Reply:
I checked the previous migration guide docs and they all have this kind of information. Let us keep it like that in this release and I will feedback to the team to see if anything we can improve.